bubbleSort.cpp

```diff
@@ -0,0 +1,12 @@
+#include "header.h"
+
+void bubbleSort(int arr[], int size) {
+
+    for(int i = 0; i < size - 1; i++) {
+        for(int j = 0; j < size - i - 1; j++) {
+            if(arr[j] > arr[j + 1]) {
+                swap(&arr[j], &arr[j + 1]);
```

Comment:
По заданию - для доступа к элементу массива используется работа с указателем. 
К примеру для условия:
```suggestion
            if( *( arr+j ) > *( arr + j + 1 ) ){
                swap( ( arr+ j ), ( arr + j + 1 ) );
```